src/modules/KYB/containers/Banner/index.types.ts

```diff
@@ -0,0 +1,10 @@
+export interface BannerProps {
+  theme: 'warning' | 'error' | 'success';
```

Comment:
no `gray`?

Reply:
Not yet, maybe later in design we'll need it